src/core/api.c

```diff
@@ -1615,6 +1615,102 @@ MsQuicGetParam(
     return Status;
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
```

Comment:
I'm wondering if we can/should require passive. I can see instances where a dispatch thread might want to add a ref. Thoughts?

Reply:
AddRef we probably can call at dispatch, but not release.